frontend-html/src/gui/Workbench/ScreenArea/FormView/FormViewEditor.tsx

```diff
@@ -367,3 +368,12 @@ export class FormViewEditor extends React.Component<{
     return this.getEditor();
   }
 }
+
+export function resolveNumericCellAlignment(customStyle: { [p: string]: string } | undefined){
+  let cellAlignment = new CellAlignment(false, "Number", customStyle);
+  const style = customStyle ?Object.assign({}, customStyle) :{};
+  style["paddingRight"] = cellAlignment.paddingRight - 11 + "px";
```

Comment:
Shouldn't there be a constant here at least? Shouldn't this be managed by assigning a different class and all the rest set up in a css?

Reply:
This method reuses code handling the custom table cell style. I guess we wanted to have the possibility to set any style the user enters into the model and that is why the style is set directly in the code, not in css.

As for the constant, realized the line with the hardcoded value is not necessary...